src/GitHub.InlineReviews/Commands/InlineCommentNavigationCommand.cs

```diff
@@ -217,7 +218,10 @@ protected IReadOnlyList<ITagInfo> GetTags(IEnumerable<ITextView> textViews)
         /// <summary>
         /// Shows the inline comments for the specified tag in a peek view.
         /// </summary>
+        /// <param name="textView"></param>
         /// <param name="tag"></param>
+        /// <param name="parameter"></param>
+        /// <param name="allTextViews"></param>
```

Comment:
`The additional text views when showing a comment in a diff view` or something?

Reply:
`The full list of text views`